docs/running-examples.rst

```diff
@@ -121,3 +138,92 @@ Add IPv4 address to you interface:
 .. warning::
 
    ``192.169.100.100/24`` and ``eth0`` are just examples!
+
+Setting up Zynq ARM CPU
+------------------------
+
+Zynq FPGAs include an ARM CPU. This guide instructs on setting up U-boot to run Linux, load bitstreams and control the Programmable Logic through the ARM CPU.
+
+.. _prepare-sd:
+
+Prepare SD card
+~~~~~~~~~~~~~~~
+
+#. Format the SD card by following the `official guide <https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/18842385/How+to+format+SD+card+for+SD+boot>`_.
+
+#. Download and extract pre-built U-boot images
+
+   .. code-block:: bash
+
+      mkdir uboot-linux-images
+      pushd uboot-linux-images
+      wget -qO- https://github.com/SymbiFlow/symbiflow-xc7z-automatic-tester/releases/download/v1.0.0/uboot-linux-images.zip | bsdtar -xf-
+      popd
+
+#. Copy boot images
+
+   .. code-block:: bash
+
+      cp uboot-linux-images/boot/* /path/to/mountpoint/boot/
+      sync
+
+#. Copy rootfs
+
+   .. code-block:: bash
+
+      wget -qO- http://de5.mirror.archlinuxarm.org/os/ArchLinuxARM-armv7-latest.tar.gz | sudo tar -xvzC /path/to/mountpoint/root
+      sudo cp -a uboot-linux-images/rootfs/* /path/to/mountpoint/root/root/
```

Comment:
is the destination path correct? I mean above you have `/path/to/mountpoint/boot/` and for rootfs `/path/to/mountpoint/root/root/` - seems one level too deep

Reply:
Yes, the first root is the root file system, while the second one is the root directory within the root filesystem.

E.g. the first `root` contains:
```
bin  boot  lost+found  media  mnt  nix  opt  proc  root ...
```